pubspec.yaml

```diff
@@ -1,14 +1,15 @@
 name: cli_util
-version: 0.2.1-dev
+version: 0.3.0-nullsafety
```

Comment:
Is the convention here `0.3.0-nullsafety` or `0.3.0-nullsafety.1`?

Reply:
No idea to be honest!

Reply:
The first place I looked has *both* conventions:

https://github.com/dart-lang/path/blob/7dedbbe99e76f7cd154e4ac26db3e9a183c3525b/pubspec.yaml#L14-L15

🤷 

@natebosch: any opinion?

Reply:
Offline conversation has lead to `0.3.0-nullsafety` as the preferred convention.